index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Embedding Images and Videos</title>
+  </head>
+  <body>
+    <header>
+      <nav class="d-flex">
+        <a class="icons" href="http://www.youtube.com">
+          <img src="images/youtube.svg" alt="youtube-icon" />
+        </a>
+        <form>
+          <input type="text" id="search-field" placeholder="search" />
+          <button id="search-btn" type="submit">Search</button>
+        </form>
+
+        <div id="nav-links-container" class="d-flex">
+          <img class="icons" src="images/video-camera.svg" alt="video-camera" />
+          <img class="icons" src="images/menu.svg" alt="menu" />
+          <img class="icons" src="images/notification.svg" alt="notification" />
+          <img id="avatar" class="icons" src="images/user.svg" alt="user" />
+        </div>
+      </nav>
+    </header>
+    <main id="main-container">
+      <section id="video-section-wrapper">
+        <section id="video-container">
+          <iframe
+            src="https://www.youtube.com/embed/V74l_zS1x8E"
+            frameborder="0"
+            allow="accelerometer; autoplay; encrypted-media; gyroscope;
+          picture-in-picture"
+            allowfullscreen
+          >
+          </iframe>
+          <section>
+            <div id="title-container">
+              <h1>Contributing to Open Source Part I: The Easy Way</h1>
+              <div class="d-flex">
+                <p id="views">
+                  57 231 vues<time datetime="2014-02-24"
+                    >• 27 February, 2014</time
+                  >
+                </p>
+                <ul class="d-flex">
+                  <li class="icons">
+                    <img src="images/like.svg" alt="like" /> 422
+                  </li>
+                  <li class="icons">
+                    <img src="images/dislike.svg" alt="dislike" />18
+                  </li>
+                  <li class="icons">
+                    <img src="images/share.svg" alt="share" /> share
+                  </li>
+                </ul>
```

Comment:
[Required] Share icon should be a button but it is an item only in your layout. [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_share_requirement.png) of the requirement.

[Required] Thumbnail images of other videos are not showing.I am using [githack](https://raw.githack.com/) to check the project.  Have a look :point_down: 
![Screenshot from 2020-06-04 07-37-30](https://user-images.githubusercontent.com/37747619/83707912-ddef8f80-a63a-11ea-9b6b-b6829cf5cd23.png)